internal/common/pgkeyvalue/pgkeyvalue_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"golang.org/x/exp/maps"
-	"k8s.io/apimachinery/pkg/util/clock"
+	"k8s.io/utils/clock"
```

Comment:
This doesn't seem to be the right change, but something like this needs to happen. I got errors when using the apimachinery code for it.